src/components/EmployeesList.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { withRouter } from 'react-router-dom';
+import { getDepartmentById, getEmployeeByDepartmentId } from '../api/api';
+
+import Employee from './Employee';
+import Loader from './Loader';
+
+class EmployeesList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
The same here, constructor is no needed, and state may be called as just property